check_typehints.py

```
@@ -0,0 +1,68 @@
+#!/usr/bin/python3
+
+import ast
+import os
+import os.path
+import sys
+
+import astor
+
+
+class CheckTypeHints(ast.NodeTransformer):
+    IGNORE_PARAMETER_NAMES = {"self", "cls", "klass"}
```

Ruff (0.12.2)

Mutable class attributes should be annotated with `typing.ClassVar`

(RUF012)